src/components/chips/js/contactChipsAutocompleteScope.js

```diff
@@ -0,0 +1,29 @@
+angular
+    .module('material.components.chips')
+    .directive('mdContactChipsAutocompleteScope', MdContactChipsAutocompleteScope);
+
+function MdContactChipsAutocompleteScope ($compile, $mdUtil) {
+  return {
+    restrict: 'A',
+    link:     postLink,
+    scope:    false
+  };
+
+  function postLink (scope, element, attr) {
+    // Since the autocomplete popup is appended to the body, we cannot traverse up the DOM looking
+    // for the proper controller and grabbing it's scope, so we must go WAY up to get the proper
+    // scope; create a new child scope based off of the parent, and copy the item over on changes
+    var newScope = scope.$parent.$parent.$parent.$new();
```

Comment:
Unfortunately, I already tried that as I too thought it would be cleaner. However, the `mdChips` controller isn't available because the element is no longer in the same spot in the DOM (it gets `detach()`ed and moved) so it can't find it.

If there is another way to get that controller, I'll all for it.